ajax/libs/inputmask-multi/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "inputmask-multi",
+  "filename": "jquery.inputmask-multi.min.js",
+  "version": "1.2.0",
+  "title": "Inputmask Multi",
+  "description": "Inputmask Multi is a plugin which allows to select a mask of input basing on the beginning of entered data (such as an international phone number)",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/andr-04/inputmask-multi.git"
+  },
+  "author": {
+    "name": "Andrey Egorov",
+    "url": "https://github.com/andr-04"
+  },
+  "dependencies": {
+    "jquery": ">=1.5",
+    "jquery.inputmask": ">=3.2.0"
+  },
+  "keywords": [
+    "inputmask",
+    "phone",
+    "phonenumber",
+    "list",
+    "multi",
+    "several",
+    "mask",
+    "selection",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "homepage": "https://github.com/andr-04/inputmask-multi",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/andr-04/inputmask-multi.git",
+    "basePath": "",
+    "files": [
+      "js/jquery.inputmask-multi*"
+    ]
```

Comment:
If this lib was also put on npm, I think you can use npm auto-update, that would be better.
And the ```homepage``` should be http://andr-04.github.io/inputmask-multi/